conan/tools/cmake/presets.py

```diff
@@ -200,6 +208,7 @@ def write_cmake_presets(conanfile, toolchain_file, generator, cache_variables,
 
     preset_content = json.dumps(data, indent=4)
     save(preset_path, preset_content)
+    ConanOutput(str(conanfile)).info("CMakeToolchain generated: CMakePresets.json")
```

Comment:
I'm guessing these are `ConanFileInterface`s and that's why they can't just call `conanfile.output...`, right?

Reply:
No, good catch, I think this might be a full conanfile, this might be replaced by ``conanfile.output.info``